changelog.md

```diff
@@ -1,9 +1,15 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
-## [0.2.2] - TBA
+## [0.3.0] - 2022-05-10
+
+### Added
+- New methods in configuration-api v3.5 for tags management: `create_tag`, `list_tags`, `update_tag`, `delete_tag`.
```

Comment:
Please also mention new method in Reports API.

Reply:
What method ?